keras_cv/layers/preprocessing/random_mix_up.py

```diff
@@ -0,0 +1,64 @@
+import tensorflow as tf
+import tensorflow.keras as keras
+from tensorflow.keras import layers
+
+
+class RandomMixUp(layers.Layer):
```

Comment:
Please add a class dosctring with `Args:` description and a usage example

Reply:
Done